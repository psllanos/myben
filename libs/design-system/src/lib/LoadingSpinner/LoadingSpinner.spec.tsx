import { render, screen } from '@testing-library/react'
import { LoadingSpinner } from './'

describe('LoadingSpinner', () => {
    it('should render properly', () => {
        const component = render(<LoadingSpinner />)

        expect(screen.getByRole('img')).toBeInTheDocument()
        expect(component).toMatchSnapshot()
    })
})
