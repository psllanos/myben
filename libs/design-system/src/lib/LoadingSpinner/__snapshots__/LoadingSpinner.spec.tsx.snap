// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner should render properly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        aria-label="loading"
        height="50"
        role="img"
        viewBox="0 0 39 25"
        width="50"
      >
        <g
          fill="#34363A"
        >
          <path
            d="m13.948 0.43726h-5.0748c-1.246 0-2.256 1.0137-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0748c1.246 0 2.256-1.0137 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641z"
          />
          <path
            d="m29.71 0.43726h-5.0748c-1.2459 0-2.256 1.0137-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0748c1.246 0 2.256-1.0137 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641z"
          />
          <path
            d="m16.332 6.9539h-9.6145c-1.246 0-2.256 1.0137-2.256 2.2641s1.01 2.2641 2.256 2.2641h9.6145c1.2459 0 2.256-1.0137 2.256-2.2641s-1.0101-2.2641-2.256-2.2641z"
          />
          <path
            d="m22.872 11.481h9.0005c1.246 0 2.256-1.0137 2.256-2.2642 0-1.2504-1.01-2.2641-2.256-2.2641h-9.0005c-1.246 0-2.2561 1.0137-2.2561 2.2641 0 1.2504 1.0101 2.2642 2.2561 2.2642z"
          />
          <path
            d="m7.5807 20.183h-5.2582c-1.246 0-2.256 1.0137-2.256 2.2641 0 1.2505 1.0101 2.2641 2.256 2.2641h5.2582c1.246 0 2.256-1.0136 2.256-2.2641 0-1.2504-1.0101-2.2641-2.256-2.2641z"
          />
          <path
            d="m31.099 24.71h5.2583c1.2459 0 2.256-1.0137 2.256-2.2641s-1.0101-2.2641-2.256-2.2641h-5.2583c-1.2459 0-2.256 1.0137-2.256 2.2641s1.0101 2.2641 2.256 2.2641z"
          />
          <path
            d="m21.466 20.167h-4.2804c-1.246 0-2.256 1.0137-2.256 2.2641s1.01 2.2641 2.256 2.2641h4.2804c1.246 0 2.256-1.0137 2.256-2.2641s-1.01-2.2641-2.256-2.2641z"
          />
          <path
            d="m9.5383 13.666h-5.0635c-1.246 0-2.256 1.0136-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0635c1.2459 0 2.256-1.0137 2.256-2.2641 0-1.2505-1.0101-2.2641-2.256-2.2641z"
          />
          <path
            d="m29.054 18.194h5.0635c1.246 0 2.256-1.0136 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641h-5.0635c-1.2459 0-2.256 1.0137-2.256 2.2641 0 1.2505 1.0101 2.2641 2.256 2.2641z"
          />
          <path
            d="m23.094 13.652h-7.5347c-1.246 0-2.2561 1.0137-2.2561 2.2641 0 1.2505 1.0101 2.2641 2.2561 2.2641h7.5347c1.246 0 2.2561-1.0136 2.2561-2.2641 0-1.2504-1.0101-2.2641-2.2561-2.2641z"
          />
        </g>
        <g
          class="transition-opacity duration-[50ms] opacity-0"
        >
          <path
            d="m13.948 0.43726h-5.0748c-1.246 0-2.256 1.0137-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0748c1.246 0 2.256-1.0137 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641z"
            fill="#4CC9F0"
          />
          <path
            d="m16.332 6.9539h-9.6145c-1.246 0-2.256 1.0137-2.256 2.2641s1.01 2.2641 2.256 2.2641h9.6145c1.2459 0 2.256-1.0137 2.256-2.2641s-1.0101-2.2641-2.256-2.2641z"
            fill="#4361EE"
          />
          <path
            d="m7.5807 20.183h-5.2582c-1.246 0-2.256 1.0137-2.256 2.2641 0 1.2505 1.0101 2.2641 2.256 2.2641h5.2582c1.246 0 2.256-1.0136 2.256-2.2641 0-1.2504-1.0101-2.2641-2.256-2.2641z"
            fill="#F72585"
          />
          <path
            d="m9.5383 13.666h-5.0635c-1.246 0-2.256 1.0136-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0635c1.2459 0 2.256-1.0137 2.256-2.2641 0-1.2505-1.0101-2.2641-2.256-2.2641z"
            fill="#7209B7"
          />
        </g>
        <g
          class="transition-opacity duration-[50ms] opacity-0"
        >
          <path
            d="m13.948 0.43726h-5.0748c-1.246 0-2.256 1.0137-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0748c1.246 0 2.256-1.0137 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641z"
            fill="#4CC9F0"
          />
          <path
            d="m16.332 6.9539h-9.6145c-1.246 0-2.256 1.0137-2.256 2.2641s1.01 2.2641 2.256 2.2641h9.6145c1.2459 0 2.256-1.0137 2.256-2.2641s-1.0101-2.2641-2.256-2.2641z"
            fill="#4361EE"
          />
          <path
            d="m21.466 20.167h-4.2804c-1.246 0-2.256 1.0137-2.256 2.2641s1.01 2.2641 2.256 2.2641h4.2804c1.246 0 2.256-1.0137 2.256-2.2641s-1.01-2.2641-2.256-2.2641z"
            fill="#F72585"
          />
          <path
            d="m23.094 13.652h-7.5347c-1.246 0-2.2561 1.0137-2.2561 2.2641 0 1.2505 1.0101 2.2641 2.2561 2.2641h7.5347c1.246 0 2.2561-1.0136 2.2561-2.2641 0-1.2504-1.0101-2.2641-2.2561-2.2641z"
            fill="#7209B7"
          />
          <path
            d="m29.71 0.43726h-5.0748c-1.2459 0-2.256 1.0137-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0748c1.246 0 2.256-1.0137 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641z"
            fill="#4CC9F0"
          />
          <path
            d="m22.872 11.481h9.0005c1.246 0 2.256-1.0137 2.256-2.2642 0-1.2504-1.01-2.2641-2.256-2.2641h-9.0005c-1.246 0-2.2561 1.0137-2.2561 2.2641 0 1.2504 1.0101 2.2642 2.2561 2.2642z"
            fill="#4361EE"
          />
        </g>
        <g
          class="transition-opacity duration-[50ms] opacity-0"
        >
          <path
            d="m29.71 0.43726h-5.0748c-1.2459 0-2.256 1.0137-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0748c1.246 0 2.256-1.0137 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641z"
            fill="#4CC9F0"
          />
          <path
            d="m22.872 11.481h9.0005c1.246 0 2.256-1.0137 2.256-2.2642 0-1.2504-1.01-2.2641-2.256-2.2641h-9.0005c-1.246 0-2.2561 1.0137-2.2561 2.2641 0 1.2504 1.0101 2.2642 2.2561 2.2642z"
            fill="#4361EE"
          />
          <path
            d="m29.054 18.194h5.0635c1.246 0 2.256-1.0136 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641h-5.0635c-1.2459 0-2.256 1.0137-2.256 2.2641 0 1.2505 1.0101 2.2641 2.256 2.2641z"
            fill="#7209B7"
          />
          <path
            d="m31.099 24.71h5.2583c1.2459 0 2.256-1.0137 2.256-2.2641s-1.0101-2.2641-2.256-2.2641h-5.2583c-1.2459 0-2.256 1.0137-2.256 2.2641s1.0101 2.2641 2.256 2.2641z"
            fill="#F72585"
          />
        </g>
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      aria-label="loading"
      height="50"
      role="img"
      viewBox="0 0 39 25"
      width="50"
    >
      <g
        fill="#34363A"
      >
        <path
          d="m13.948 0.43726h-5.0748c-1.246 0-2.256 1.0137-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0748c1.246 0 2.256-1.0137 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641z"
        />
        <path
          d="m29.71 0.43726h-5.0748c-1.2459 0-2.256 1.0137-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0748c1.246 0 2.256-1.0137 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641z"
        />
        <path
          d="m16.332 6.9539h-9.6145c-1.246 0-2.256 1.0137-2.256 2.2641s1.01 2.2641 2.256 2.2641h9.6145c1.2459 0 2.256-1.0137 2.256-2.2641s-1.0101-2.2641-2.256-2.2641z"
        />
        <path
          d="m22.872 11.481h9.0005c1.246 0 2.256-1.0137 2.256-2.2642 0-1.2504-1.01-2.2641-2.256-2.2641h-9.0005c-1.246 0-2.2561 1.0137-2.2561 2.2641 0 1.2504 1.0101 2.2642 2.2561 2.2642z"
        />
        <path
          d="m7.5807 20.183h-5.2582c-1.246 0-2.256 1.0137-2.256 2.2641 0 1.2505 1.0101 2.2641 2.256 2.2641h5.2582c1.246 0 2.256-1.0136 2.256-2.2641 0-1.2504-1.0101-2.2641-2.256-2.2641z"
        />
        <path
          d="m31.099 24.71h5.2583c1.2459 0 2.256-1.0137 2.256-2.2641s-1.0101-2.2641-2.256-2.2641h-5.2583c-1.2459 0-2.256 1.0137-2.256 2.2641s1.0101 2.2641 2.256 2.2641z"
        />
        <path
          d="m21.466 20.167h-4.2804c-1.246 0-2.256 1.0137-2.256 2.2641s1.01 2.2641 2.256 2.2641h4.2804c1.246 0 2.256-1.0137 2.256-2.2641s-1.01-2.2641-2.256-2.2641z"
        />
        <path
          d="m9.5383 13.666h-5.0635c-1.246 0-2.256 1.0136-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0635c1.2459 0 2.256-1.0137 2.256-2.2641 0-1.2505-1.0101-2.2641-2.256-2.2641z"
        />
        <path
          d="m29.054 18.194h5.0635c1.246 0 2.256-1.0136 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641h-5.0635c-1.2459 0-2.256 1.0137-2.256 2.2641 0 1.2505 1.0101 2.2641 2.256 2.2641z"
        />
        <path
          d="m23.094 13.652h-7.5347c-1.246 0-2.2561 1.0137-2.2561 2.2641 0 1.2505 1.0101 2.2641 2.2561 2.2641h7.5347c1.246 0 2.2561-1.0136 2.2561-2.2641 0-1.2504-1.0101-2.2641-2.2561-2.2641z"
        />
      </g>
      <g
        class="transition-opacity duration-[50ms] opacity-0"
      >
        <path
          d="m13.948 0.43726h-5.0748c-1.246 0-2.256 1.0137-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0748c1.246 0 2.256-1.0137 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641z"
          fill="#4CC9F0"
        />
        <path
          d="m16.332 6.9539h-9.6145c-1.246 0-2.256 1.0137-2.256 2.2641s1.01 2.2641 2.256 2.2641h9.6145c1.2459 0 2.256-1.0137 2.256-2.2641s-1.0101-2.2641-2.256-2.2641z"
          fill="#4361EE"
        />
        <path
          d="m7.5807 20.183h-5.2582c-1.246 0-2.256 1.0137-2.256 2.2641 0 1.2505 1.0101 2.2641 2.256 2.2641h5.2582c1.246 0 2.256-1.0136 2.256-2.2641 0-1.2504-1.0101-2.2641-2.256-2.2641z"
          fill="#F72585"
        />
        <path
          d="m9.5383 13.666h-5.0635c-1.246 0-2.256 1.0136-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0635c1.2459 0 2.256-1.0137 2.256-2.2641 0-1.2505-1.0101-2.2641-2.256-2.2641z"
          fill="#7209B7"
        />
      </g>
      <g
        class="transition-opacity duration-[50ms] opacity-0"
      >
        <path
          d="m13.948 0.43726h-5.0748c-1.246 0-2.256 1.0137-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0748c1.246 0 2.256-1.0137 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641z"
          fill="#4CC9F0"
        />
        <path
          d="m16.332 6.9539h-9.6145c-1.246 0-2.256 1.0137-2.256 2.2641s1.01 2.2641 2.256 2.2641h9.6145c1.2459 0 2.256-1.0137 2.256-2.2641s-1.0101-2.2641-2.256-2.2641z"
          fill="#4361EE"
        />
        <path
          d="m21.466 20.167h-4.2804c-1.246 0-2.256 1.0137-2.256 2.2641s1.01 2.2641 2.256 2.2641h4.2804c1.246 0 2.256-1.0137 2.256-2.2641s-1.01-2.2641-2.256-2.2641z"
          fill="#F72585"
        />
        <path
          d="m23.094 13.652h-7.5347c-1.246 0-2.2561 1.0137-2.2561 2.2641 0 1.2505 1.0101 2.2641 2.2561 2.2641h7.5347c1.246 0 2.2561-1.0136 2.2561-2.2641 0-1.2504-1.0101-2.2641-2.2561-2.2641z"
          fill="#7209B7"
        />
        <path
          d="m29.71 0.43726h-5.0748c-1.2459 0-2.256 1.0137-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0748c1.246 0 2.256-1.0137 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641z"
          fill="#4CC9F0"
        />
        <path
          d="m22.872 11.481h9.0005c1.246 0 2.256-1.0137 2.256-2.2642 0-1.2504-1.01-2.2641-2.256-2.2641h-9.0005c-1.246 0-2.2561 1.0137-2.2561 2.2641 0 1.2504 1.0101 2.2642 2.2561 2.2642z"
          fill="#4361EE"
        />
      </g>
      <g
        class="transition-opacity duration-[50ms] opacity-0"
      >
        <path
          d="m29.71 0.43726h-5.0748c-1.2459 0-2.256 1.0137-2.256 2.2641 0 1.2504 1.0101 2.2641 2.256 2.2641h5.0748c1.246 0 2.256-1.0137 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641z"
          fill="#4CC9F0"
        />
        <path
          d="m22.872 11.481h9.0005c1.246 0 2.256-1.0137 2.256-2.2642 0-1.2504-1.01-2.2641-2.256-2.2641h-9.0005c-1.246 0-2.2561 1.0137-2.2561 2.2641 0 1.2504 1.0101 2.2642 2.2561 2.2642z"
          fill="#4361EE"
        />
        <path
          d="m29.054 18.194h5.0635c1.246 0 2.256-1.0136 2.256-2.2641 0-1.2504-1.01-2.2641-2.256-2.2641h-5.0635c-1.2459 0-2.256 1.0137-2.256 2.2641 0 1.2505 1.0101 2.2641 2.256 2.2641z"
          fill="#7209B7"
        />
        <path
          d="m31.099 24.71h5.2583c1.2459 0 2.256-1.0137 2.256-2.2641s-1.0101-2.2641-2.256-2.2641h-5.2583c-1.2459 0-2.256 1.0137-2.256 2.2641s1.0101 2.2641 2.256 2.2641z"
          fill="#F72585"
        />
      </g>
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
